docs/articles/life-cycle/skipping-tests.rst

```diff
@@ -166,3 +166,37 @@ tests and are not available at this point.
                     tests/test_passkey.py::test_skip__lambda (ldap) SKIPPED (Server is not built with password policy support)
                     tests/test_passkey.py::test_skip__function (ldap) SKIPPED (Server is not built with password policy support)
                     tests/test_passkey.py::test_skip__function_and_reason (ldap) SKIPPED (Server is not built with password policy support)
+
+Preferred topology
+===================
+
+Sometimes, you want to maintain the flexibility to run tests against all topologies, but not always. Nightly builds and CI are examples where running a test against a single topology by default may make more sense than all topologies. This can reduce the resources required to run the tests and their execution times. All while using the same test code. This is possible by using the ``pytest.mark.preferred_topology`` marker. This marks the test with a default topology that will run, and for the comprehensive runs, "--mh-ignore-preferred-topology" will ignore this marker entirely.
```

Comment:
Keep 80 chars limit for text.

Reply:
It might be better to move this to tips and tricks section.

Reply:
I moved it to tips and tricks. I was planning on dropping it entirely. If speed and fewer resources are the goal, this **that** should be the path. The choice to skip was your recommendation, so we knew it what was being skipped, do you want me to drop it earlier, during collection?